test/intl402/Locale/invalid-tag-throws-symbol.js

```diff
@@ -2,14 +2,16 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
-esid: pending
+esid: sec-intl.locale
 description: >
     Verifies the type check on the tag argument to Intl.Locale.
 info: |
     Intl.Locale( tag [, options] )
+
     7. If Type(tag) is not String or Object, throw a TypeError exception.
 features: [Intl.Locale]
 ---*/
 
-const symbol = Symbol();
-assert.throws(TypeError, () => new Intl.Locale(symbol));
+assert.throws(TypeError, function() {
+  new Intl.Locale(Symbol());
```

Comment:
This will now incorrectly pass if the `Symbol` constructor throws for some odd reason, rather than the `Locale` constructor.